trlc/trlc.py

```diff
@@ -787,6 +787,16 @@ def main():
     if sm.process() is None:
         ok = False
 
+    # for file, parser in sm.all_files.items():
+    #     for tok in parser.lexer.tokens:
+    #         if tok.value is None:
+    #             token = tok.kind
+    #         else:
+    #             token = tok.value
+    #         print(f"Token: {token} => {tok.ast_link}")
+    #         assert tok.ast_link is not None, \
+    #             f"Token {tok} in {file} has no ast_link"
```

Comment:
Please remove dead code.

Reply:
@phiwuu: Thanks for your feedback! It might be better to wait. Reviewing the pull request could become more challenging without these lines. I've referenced them in the pull request description.

Reply:
Please remove now, and merge afterwards.